fitbenchmarking/fitbenchmarking
# 659 release minimizers bug

#### Description of Work
Fixes #659 

Fixes the failed minimizers on the releases results pages.

#### Testing Instructions

1.  Check that that there are no reports of failed minimizers in the results pages.

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
